pages/[username]/index.tsx

```diff
@@ -17,9 +20,13 @@ function Header() {
         ) : domain === 'treasury' ? (
           <Treasury />
         ) : (
-          <SSI>
-            <div /> {/*@todo-1 is this needed?*/}
-          </SSI>
+          <>
+            {!loading ? (
+              <SSI>
+                <div /> {/*@todo-1 is this needed? : Yes, it needs a blank component to avoid eror on typescript*/}
```

Comment:
great, thank you @DevPH-Wistkey! In this case that there is an answer from you, let's just remove the @todo-1 and leave the rest: "is this needed? : Yes, it needs a blank component to avoid eror on typescript"